themes/src/main/resources/theme/keycloak.v2/login/register.ftl

```diff
@@ -46,5 +45,39 @@
             </div>
 
         </form>
+        <script type="module">
+            import { validatePassword } from "${url.resourcesPath}/js/password-policy.js";
+
+            const template = `
+            <div class="pf-v5-c-form__helper-text" aria-live="polite">
+                <div class="pf-v5-c-helper-text">
+                    <div class="pf-v5-c-helper-text__item pf-m-error">
+                        <ul class="pf-v5-c-helper-text__item-text">
+                            {errors}
+                        </ul>
+                    </div>
+                </div>
+            </div>`
+
+            const activePolicies = [
+                { length: { value: ${passwordPolicies.length!-1}, error: "${msg('invalidPasswordMinLengthMessage')}"} },
```

Comment:
Are these messages escaped? Seems like an easy place to inject a malicious script.

Reply:
it's an error message from the server that is rendered on the server you can only change it by having access to the said server

Reply:
not only that, but freemarker escapes it and there are tests that check that

Reply:
Great, just making sure, it's been a while since I touched this stuff.